protos/flyteidl/admin/matchable_resource.proto

```diff
@@ -58,6 +61,29 @@ message ExecutionClusterLabel {
   string value	= 1;
 }
 
+message PluginOverride {
```

Comment:
lets have dome documentation on the message structure itself?

Reply:
good point, done